types/src/on_chain_config/timed_features.rs

```diff
@@ -115,7 +118,14 @@ impl TimedFeatureFlag {
                 .with_ymd_and_hms(2025, 3, 11, 17, 0, 0)
                 .unwrap()
                 .with_timezone(&Utc),
-
+            (DisabledCaptureOption, TESTNET) => Los_Angeles
+                .with_ymd_and_hms(2025, 9, 15, 12, 0, 0)
+                .unwrap()
+                .with_timezone(&Utc),
+            // For testing, time set to 1 hour after the beginning of time to test the old and new behaviors in tests.
+            (DisabledCaptureOption, TESTING) => Utc.with_ymd_and_hms(1970, 1, 1, 1, 0, 0).unwrap(),
+            // For mainnet, always enable this feature.
+            (DisabledCaptureOption, MAINNET) => BEGINNING_OF_TIME,
```

Comment:
this number can't be used for release..

Reply:
which number should be used for mainnet in general? cc @georgemitenkov

Reply:
@zekun000 why? We don't have function values on so nothing is captured, so in theory this capturing is disabled since the beginning because there is no capturing?

Reply:
ah okay so the timed feature is only for testnet